src/install/package.ts

```diff
@@ -420,6 +420,12 @@ export function validateOverride (pcfg: PackageConfig, name: string) {
 }
 
 export function processPackageConfig (pcfg: PackageConfig, partial = false, registry = ''): ProcessedPackageConfig {
+
+  // Override with jspm section if it exists.
+  if (typeof pcfg.jspm === 'object') {
```

Comment:
When overriding dependencies it is important that any `jspm.dependencies` means all dependency properties are overridden.

This can be captured with the following logic:

```js
if (pcfg.jspm && (pcfg.jspm.dependencies || pcfg.jspm.devDependencies || pcfg.jspm.peerDependencies || pcfg.jspm.optionalDependencies)) {
  pcfg.dependencies = pcfg.jspm.dependencies || {};
  pcfg.devDependencies = pcfg.jspm.devDependencies || {};
  pcfg.peerDependencies = pcfg.jspm.peerDependencies || {};
  pcfg.optionalDependencies = pcfg.jspm.optionalDependencies || {};
}
```

Happy to merge with that change.